### ISSUE TITLE
columns columns columns - quite an issue

### ISSUE THREAD

**Issue by dasigna** [pojome/elementor#1730]
i know, this has been posted before.
but it really turns out to be an issue when trying to set up the exact width of columns.
did realize this a long time, but managed to 'live with it' - now that i had to teach someone how to set up content this came to me again (and sadly not in the best way...!).

when dragging columns - lets say three in a row - its really a mess when trying to get the rightmost to the desired (exact) width. there are no percentages shown for this one when dragging. theres always the percentage of the one left to the cursor shown - but you cant drag the rightmost section from the outside...

another one is that the shown percentages are covered by the row and section labels as soon as the dragged rows are going under a certain percentage.

third one is, (but not that hard) you obviously cant get to exact 100 percent alltogether, and hardly or never match the min amount of 10%.

so to sum up:
this really needs to be improved! (dont want to drive new users insane with this again ;-) )
eighter by making the percentages more visible and reliable, or (maybe even better) by direct access to input percentages manually on demand.

tiny little thing, but quite disturbing sometimes :-)

dont mind - thanks for all the great work and keep it up!

**Comment by carasmo** [comment]
This is the natural way LTR works. The right most column is the remaining width of the two previous columns, so if you want a right column that is 40% of the container, then the two previous need to add up to 60%. You can see the width of the right column by hovering over it. If you want to work RTL you can use the RTL tester plugin, switch your site to RTL, adjust the left column (which is the right column) and then save and turn off the tester. I'm not one of the devs, but this would be a not going to fix.

**Comment by dasigna** [comment]
yes, yes ... agree - partly.
(and know the workarounds)

but this does not resolve the tiny little issue, that the percentages are not showing up for the right column while dragging (not to mention the covering again).

ever tried to tell this to a simple user? :-)

**Comment by carasmo** [comment]
I see the percentages for all columns when I hover. 
![screen shot 2017-05-16 at 2 38 09 pm](https://cloud.githubusercontent.com/assets/1124594/26122427/6bfba3a0-3a45-11e7-9207-340be4aa2341.png)

Simple users still need to understand the math even if you see their eyes go dull, which happens a lot unfortunately.

**Comment by dasigna** [comment]
yeah, see them too on hover, but not while dragging! :)

**Comment by carasmo** [comment]
Ahh, I see what you mean. At least they should add fields to add the percentage in, that would be handy.

**Comment by pingram3541** [comment]
I'd like to add that my frustration with columns is always trying to get just the right amount as "dragging" is the only way vs a manual method which I've requested a long time ago.  I've since had to resort to getting a super high dpi mouse with a button that I can reduce the sensitivity quickly so I can drag and get that exact % I am looking for and then go back to my regular mouse speed.

Therefore I agree with @dasigna, the columns "width" setting's user experience should be better scrutinized to provide easier freedom and control of the desired values than it currently allows.  I've built tons of sites since fruition and this is a common pain point for me.

**Comment by arielk** [comment]
It's already done for the upcoming version 1.5.0! (Beta soon).

See the print screen:
![upymqrm](https://cloud.githubusercontent.com/assets/1778512/26171404/87c498f4-3b4d-11e7-8ba5-8a2460f88b1a.png)

Thanks for your feedback!

Ref: #847

**Comment by dasigna** [comment]
WOW!!! thats great news - really.

you guys are amazing - thanks :-)

**Comment by pingram3541** [comment]
Yes indeed, thank you very much!!!
